### PR TITLE
Handler for simple-uploader

### PR DESCRIPTION
**Pull request by dyktek**
I just added handler for https://github.com/simple-uploader and it's working nice also with https://github.com/simple-uploader/vue-uploader

this is base on ChunksInRequestUploadHandler.php

### CODE REVIEW

**Comment by pionl** [comment]
Sweat! I would have just 2 requests to make it better:

1. Please rebase the commits into single commit without the `.idea` related files (you can reset the last commit and force push the new commit)
2. Change the `ChunksInRequestUploadHandler`  to use const (use static::KEY_CHUNK_NUMBER, etc), then excend the `ChunksInRequestUploadHandler ` in `ChunksInRequestSimpleUploadHandler` and override the const variables. This should allow using different keys for accessing data from request. Thanks to this logic will be same.

Is it clear what I'm suggesting? :)

**Comment by dyktek** [comment]
Done :) https://github.com/dyktek/laravel-chunk-upload/

**Comment by pionl** [comment]
Nice, unfortunately php 7 tests failed. Probably the `protected` ? For me it can be removed.

```
1) ChunkTests\Handler\DropZoneUploadHandlerTest::testInitWithoutRequestData
ParseError: syntax error, unexpected 'const' (T_CONST), expecting variable (T_VARIABLE)
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/ChunksInRequestUploadHandler.php:26
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/DropZoneUploadHandler.php:11
/home/travis/build/pionl/laravel-chunk-upload/tests/Handler/DropZoneUploadHandlerTest.php:24
2) ChunkTests\Handler\DropZoneUploadHandlerTest::testCanBeUsedForInvalidRequest
ParseError: syntax error, unexpected 'const' (T_CONST), expecting variable (T_VARIABLE)
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/ChunksInRequestUploadHandler.php:26
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/DropZoneUploadHandler.php:11
/home/travis/build/pionl/laravel-chunk-upload/tests/Handler/DropZoneUploadHandlerTest.php:38
3) ChunkTests\Handler\DropZoneUploadHandlerTest::testCanBeUsedForInvalidRequestPartDaata
ParseError: syntax error, unexpected 'const' (T_CONST), expecting variable (T_VARIABLE)
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/ChunksInRequestUploadHandler.php:26
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/DropZoneUploadHandler.php:11
/home/travis/build/pionl/laravel-chunk-upload/tests/Handler/DropZoneUploadHandlerTest.php:47
4) ChunkTests\Handler\DropZoneUploadHandlerTest::testCanBeUsedOnValidRequest
ParseError: syntax error, unexpected 'const' (T_CONST), expecting variable (T_VARIABLE)
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/ChunksInRequestUploadHandler.php:26
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/DropZoneUploadHandler.php:11
/home/travis/build/pionl/laravel-chunk-upload/tests/Handler/DropZoneUploadHandlerTest.php:58
5) ChunkTests\Handler\DropZoneUploadHandlerTest::testValidChunkRequest
ParseError: syntax error, unexpected 'const' (T_CONST), expecting variable (T_VARIABLE)
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/ChunksInRequestUploadHandler.php:26
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/DropZoneUploadHandler.php:11
/home/travis/build/pionl/laravel-chunk-upload/tests/Handler/DropZoneUploadHandlerTest.php:68
6) ChunkTests\Handler\DropZoneUploadHandlerTest::testValidChunkFinishRequest
ParseError: syntax error, unexpected 'const' (T_CONST), expecting variable (T_VARIABLE)
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/ChunksInRequestUploadHandler.php:26
/home/travis/build/pionl/laravel-chunk-upload/src/Handler/DropZoneUploadHandler.php:11
/home/travis/build/pionl/laravel-chunk-upload/tests/Handler/DropZoneUploadHandlerTest.php:86
```

**Comment by dyktek** [comment]
done :)

File: src/Handler/ChunksInRequestSimpleUploadHandler.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * @package Pion\Laravel\ChunkUpload\Handler
+ */
+class ChunksInRequestSimpleUploadHandler extends ChunksInRequestUploadHandler
+{
+    /**
+     * Key for number of sending chunk
+     * @static string
+     */
+    protected const KEY_CHUNK_NUMBER = 'chunkNumber';
```
Comment (pionl) [inline src/Handler/ChunksInRequestSimpleUploadHandler.php]: Remove please too :)

File: src/Handler/ChunksInRequestSimpleUploadHandler.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    /**
+     * Key for number of all chunks
+     * @static string
+     */
+    protected const KEY_ALL_CHUNKS = 'totalChunks';
```
Comment (pionl) [inline src/Handler/ChunksInRequestSimpleUploadHandler.php]: Remove please too :)

**Comment by pionl** [comment]
Thanks, last request (thank you for the contribution, it's awesome!), will merge today.

**Comment by dyktek** [comment]
done

**Pull request merged**